packages/web/src/scss/components/TextArea/index.html

```diff
@@ -0,0 +1,75 @@
+    <label for="textareaMessage" class="TextArea__label">Label</label>
+    <textarea id="textareaMessage" class="TextArea__input" placeholder="Placeholder"></textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <div class="TextArea">
+    <label for="textareaFilled" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaFilled" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+  </div>
+  <div class="TextArea">
+    <label for="textareaFilledMessage" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaFilledMessage" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <div class="TextArea TextArea--error">
+    <label for="textareaError" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaError" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+  </div>
+  <div class="TextArea TextArea--error">
+    <label for="textareaErrorMessage" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaErrorMessage" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <div class="TextArea TextArea--success">
+    <label for="textareaSuccess" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaSuccess" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <div class="TextArea TextArea--warning">
+    <label for="textareaWarning" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaWarning" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <div class="TextArea TextArea--disabled">
+    <label for="textareaDisabled" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaDisabled" class="TextArea__input" placeholder="Placeholder" disabled></textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <div class="TextArea TextArea--disabled">
+    <label for="textareaDisabledFilled" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea
+      id="textareaDisabledFilled"
+      class="TextArea__input"
+      placeholder="Placeholder"
+      disabled
+   >Filled</textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+  <div class="TextArea TextArea--fluid">
+    <label for="textareaFluid" class="TextArea__label TextArea__label--required">Label</label>
+    <textarea id="textareaFluid" class="TextArea__input" placeholder="Placeholder">Filled</textarea>
+    <div class="TextArea__message">Message</div>
+  </div>
+
+  <div style="display: flex; gap: 1rem; align-items: flex-end;">
```

Comment:
flex-end?

Reply:
Because of this: 
![image](https://user-images.githubusercontent.com/33354913/201929871-1cbd604a-f418-49aa-b64a-99cf5ec4b405.png)

When users enlarge the textarea height, the don't want to scroll to the top of the element to click the submit button. The truth is that default alignment would be better when the message is visible. Hmm, I'll remove it.

Reply:
Yop, because of this: 
![image](https://user-images.githubusercontent.com/33354913/201964054-9aaaab11-5b4c-4509-b34c-520b71408b1d.png).

It's not so robust as your little hack in Jobs, but for the documentation purpose it's good enough. I hope. 🙂